app/views/shared/tags/_curated_links_preview.html.erb

```diff
@@ -26,6 +34,14 @@
 <h4>Uncurated items <em>(not shown to user)</em></h4>
 <ul>
 <% tag.uncurated_tagged_documents.each do |item| %>
-  <li><%= item.title %></li>
+  <li class="tagged-document">
+    <% if item.base_path %>
+      <a href=<%= Plek.new.find('content-tagger') + '/taggings/lookup' + item.base_path %>>
```

Comment:
If we linked to the tagging page without content-id we would not need to link each document but we could just add a single link in the page. With Mat we thought it would have been really useful to be able to click on a link in collections-publisher and be linked straightaway to its page on content-tagger. What do you think?